app/graphql/types/query_type.rb

```diff
@@ -27,6 +27,7 @@ def books
         .joins(:user_books)
         .where("user_books.status = 0")
         .take(20)
+        .uniq
```

Comment:
Do we want to use distinct instead to keep it as a sql query? 
https://stackoverflow.com/questions/39575398/rails-uniq-vs-distinct

Reply:
good call. I changed it to use .distinct instead of .uniq. Thank you!